release-notes/System.ServiceModel.Primitives.md

```diff
@@ -0,0 +1,15 @@
+# System.ServiceModel.Primitives
+
+## Information
+
+* [NuGet Package](https://nuget.org/packages/System.ServiceModel.Primitives)
+* [Report an issue](http://github.com/dotnet/wcf/issues/new)
+* [Source](https://github.com/dotnet/wcf/tree/master/src)
+
+## Version History
+
+### 4.1.0-beta-23516 
+
+* [Release tag](https://github.com/dotnet/wcf/tree/v1.0.0-rc1)
+* [Supported features](https://github.com/dotnet/wcf/blob/master/release-notes/SupportedFeatures-v1.0.0-rc1.md)
+* Adds new public API surface area to enable use of X509Certificate2
```

Comment:
Question: is the plan to put "what's new" information in each package md or a central release note?